### ISSUE TITLE
How to copy the lat and long from LocateButton in the EDIT window for a point directly in cmv?

### ISSUE THREAD

**Issue by GilbertLasnier** [cmv/cmv-app#867]
I work yesterday with a Samsung phone the precision is 3m, I am very surprise, I want to use cmv the EDIT SECTION to add point, I do that many times but I want to collect the coordinate from the device directly in cmv, DO YOU THINK ITS WILL BE POSSIBLE??
How to USE THE LOCATE BUTTON TO COPY THE LAT AND LONG IN the EDIT
for a point or automatically like Collector, 
I just copy the code of the locate button!
![image](https://user-images.githubusercontent.com/6784515/48072378-5ca7f080-e1aa-11e8-9ad3-3871471f2343.png)

locateButton: {
   include: true,
   id: 'locateButton',
   type: 'domNode',
   path: 'gis/dijit/LocateButton',
   srcNodeRef: 'locateButton',
   options: {
      map: true,
      scale: 2400 /* https://developers.arcgis.com/javascript/jsapi/locatebutton-amd.html#scale */
      // show GPS data in a InfoTemplate
      publishGPSPosition: true,
      highlightLocation: true,
      // move map when new GPS location is obtained
      useTracking: true,
      // uncomment the next block if you want the user's location to appear on the map when app loads
      // end-user may have to allow permissions within the browser to interact with app's location request
      /*startup: function(){
            this.locate();
      },*/
      geolocationOptions: {
         maximumAge: 0,
         timeout: 15000,
         enableHighAccuracy: true
      }
   }
}

Also how to put more then one point in the Measurement tools?
![image](https://user-images.githubusercontent.com/6784515/48072298-36825080-e1aa-11e8-8ee3-d95e3da386f1.png)

| CMV Version 2.0|

**Comment by tmcgee** [comment]
@GilbertLasnier Everything you ask for is possible within the CMV architectural framework but not directly with the 3 widgets you mention: Locate Button, Editor and Measurement. These are Esri widgets so not very open to modification or customization. A custom widget or set of widgets would probably be the best approach. Specifically, editing is particularly complex and most often tied to a particular need. 

I know what you are asking for is possible because I have created a number of widgets like this for my customers. Unfortunately, these widgets I have created are tied to their specific internal workflows, in some cases proprietary and so I cannot share them on GitHub.

You mention Collector. For what it is worth, my customers have had good success with mixing data collected by Collector with CMV applications.

Hope this helps.
